app/models/credit.rb

```diff
@@ -38,4 +40,17 @@ def item_name
     # TODO: change it depending on invoice template
     credit_note.invoice.number
   end
+
+  def invoice_coupon_display_name
+    return nil if applied_coupon.blank?
+
+    coupon = applied_coupon.coupon
+    suffix = if coupon.percentage?
+               "#{sprintf("%.2f", applied_coupon.percentage_rate)}%"
+             else
+               applied_coupon.amount.format
+             end
```

Comment:
Layout/EndAlignment: `end` at 52, 13 is not aligned with `suffix = if` at 48, 4.